src/main/java/org/apache/commons/lang3/math/NumberUtils.java

```diff
@@ -32,43 +32,43 @@
 public class NumberUtils {
 
     /** Reusable Long constant for zero. */
-    public static final Long LONG_ZERO = Long.valueOf(0L);
+    public static final Long LONG_ZERO = 0L;
     /** Reusable Long constant for one. */
-    public static final Long LONG_ONE = Long.valueOf(1L);
+    public static final Long LONG_ONE = 1L;
     /** Reusable Long constant for minus one. */
-    public static final Long LONG_MINUS_ONE = Long.valueOf(-1L);
+    public static final Long LONG_MINUS_ONE = -1L;
     /** Reusable Integer constant for zero. */
-    public static final Integer INTEGER_ZERO = Integer.valueOf(0);
+    public static final Integer INTEGER_ZERO = 0;
     /** Reusable Integer constant for one. */
-    public static final Integer INTEGER_ONE = Integer.valueOf(1);
+    public static final Integer INTEGER_ONE = 1;
     /** Reusable Integer constant for two */
-    public static final Integer INTEGER_TWO = Integer.valueOf(2);
+    public static final Integer INTEGER_TWO = 2;
     /** Reusable Integer constant for minus one. */
-    public static final Integer INTEGER_MINUS_ONE = Integer.valueOf(-1);
+    public static final Integer INTEGER_MINUS_ONE = -1;
     /** Reusable Short constant for zero. */
-    public static final Short SHORT_ZERO = Short.valueOf((short) 0);
+    public static final Short SHORT_ZERO = (short) 0;
     /** Reusable Short constant for one. */
-    public static final Short SHORT_ONE = Short.valueOf((short) 1);
+    public static final Short SHORT_ONE = (short) 1;
     /** Reusable Short constant for minus one. */
-    public static final Short SHORT_MINUS_ONE = Short.valueOf((short) -1);
+    public static final Short SHORT_MINUS_ONE = (short) -1;
     /** Reusable Byte constant for zero. */
-    public static final Byte BYTE_ZERO = Byte.valueOf((byte) 0);
+    public static final Byte BYTE_ZERO = (byte) 0;
     /** Reusable Byte constant for one. */
-    public static final Byte BYTE_ONE = Byte.valueOf((byte) 1);
+    public static final Byte BYTE_ONE = (byte) 1;
     /** Reusable Byte constant for minus one. */
-    public static final Byte BYTE_MINUS_ONE = Byte.valueOf((byte) -1);
+    public static final Byte BYTE_MINUS_ONE = (byte) -1;
     /** Reusable Double constant for zero. */
-    public static final Double DOUBLE_ZERO = Double.valueOf(0.0d);
+    public static final Double DOUBLE_ZERO = 0.0d;
     /** Reusable Double constant for one. */
-    public static final Double DOUBLE_ONE = Double.valueOf(1.0d);
+    public static final Double DOUBLE_ONE = 1.0d;
```

Comment:
Hi @arturobernalg 
Thank you for your PR.
-1: For low-level libraries like most Apache Commons libraries, I like to keep boxing and unboxing explicit to make it easier to understand where objects are created and unwrapped.